dags/taar_weekly.py

```diff
@@ -14,14 +17,28 @@
 taar_ensemble_cluster_name = "dataproc-taar-ensemble"
 taar_gcpdataproc_conn_id = "google_cloud_airflow_dataproc"
 
+TAAR_BIGTABLE_INSTANCE_ID = Variable.get("taar_bigtable_instance_id")
+TAAR_ETL_STORAGE_BUCKET = Variable.get("taar_etl_storage_bucket")
+TAAR_PROFILE_PROJECT_ID = Variable.get("taar_gcp_project_id")
+TAAR_DATAFLOW_SUBNETWORK = Variable.get("taar_dataflow_subnetwork")
+
+TAAR_ETL_CONTAINER_IMAGE = (
+    "gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.4.7"
+)
+
+
 default_args_weekly = {
     "owner": "vng@mozilla.com",
+    "email": [
+        "vng@mozilla.com",
+        "mlopatka@mozilla.com",
+        "telemetry-alerts@mozilla.com",
+    ],
     "depends_on_past": False,
     "start_date": datetime(2020, 4, 4),
-    "email": ["telemetry-alerts@mozilla.com"],
     "email_on_failure": True,
     "email_on_retry": True,
-    "retries": 2,
+    "retries": 0,
```

Comment:
We don't want to retry on failure?

Reply:
I don't think there's a point to retry.  

I've come to mistrust the airflow emails as the email notifications come on every retry attempt.    I would rather have no retries so that every failure email is treated as something that should be important enough to look at.

Reply:
@jasonthomas Maybe we should revisit the retry count after a couple weeks of having this job in production?